### ISSUE TITLE
Ant Design X 1.0 is out

### ISSUE THREAD

**Issue by zombieJ** [ant-design/x#206]
[中文](https://github.com/ant-design/x/issues/205) | English

 ## Introduction

Ant Design X is a brand new AGI component library from Ant Design, designed to help developers more easily develop AI product user interfaces. Building on Ant Design, Ant Design X further expands the design specifications for AI products, offering developers more powerful tools and resources. We look forward to driving the development of AI technology with you!

+ Official Website: [https://x.ant.design/index-cn](https://x.ant.design/index-cn)
+ GitHub Repo: [https://github.com/ant-design/x](https://github.com/ant-design/x)
+ npm: `@ant-design/x`

![Home Page](https://github.com/user-attachments/assets/769cf752-048e-47ec-9927-90d96677aab0)

![Demos](https://github.com/user-attachments/assets/47f2b3e7-464f-40b2-835f-22d2e7e35be3)

## Design Philosophy
Ant Design X builds a systematic theory and methodology for AI product design— the "RICH Design Paradigm"— to help everyone understand what an AI product interface is and create an excellent AI product experience.

![RICH](https://github.com/user-attachments/assets/f11290ec-730f-4ae2-9c45-bf2a31af702a)

This theory and methodology guide us to focus not only on the tangible interface experience but also on the intangible experience rules. It will lead us through the AI design journey, gradually identifying invisible experience rules such as intent, roles, conversations, and defining hybrid interface forms that integrate various interaction modes like natural language conversation. Want to know more? Visit the [Design Module](https://x.ant.design/docs/spec/introduce) on the official Ant Design X website!

![RICH-HOW](https://github.com/user-attachments/assets/e64925c5-ede4-419b-b69e-968f41543b97)

## Rich Components
The design paradigm is split into multiple components, giving developers great flexibility in combination. Additionally, we provide technical tools that make data consumption ready to use out of the box, enabling a single X to accomplish all front-end development work:

![Components](https://github.com/user-attachments/assets/3e520125-79e7-44d8-9eb5-87affa124cdb)

#### Combinable as Needed

![Chat](https://github.com/user-attachments/assets/b90a2dc9-a45d-4c26-85d0-ad80fb40e636)

## Future Plans
Ant Design X aims to provide a consistent user experience with Ant Design, making it easy to expand and integrate with existing projects. In the future, we will also offer style extensions for mobile design specifications, enabling developers to conduct multi-terminal development simultaneously. Stay tuned.

![Mobile](https://github.com/user-attachments/assets/4c810e81-005e-4e0c-9c33-d0f88a88b7ea)

## In Conclusion
As a community-driven open-source project, Ant Design has greatly benefited from the help of the community. We thank everyone!

We hope that Ant Design X will not only help developers improve work efficiency and reduce the learning curve but also provide consistent and high-quality user experiences through detailed design specifications and a friendly development environment, helping developers tackle various complex design needs with ease.

So, give it a try: [https://x.ant.design/](https://x.ant.design/)

**Comment by brunocoelho** [comment]
@zombieJ there's a typo on the label, it should say `Announcement`

**Comment by dameekoh** [comment]
The landing page looks very beautiful and looks highly customized version of antd. Do you have a link for code or some example of how you styled it?
